harvard-lts/etd_alma_service
# Etd 161: Add existing scripts - etd_alma_service - Read/submit

**Add existing scripts - etd_alma_service - Read/submit**
* * *

**JIRA Ticket**: https://at-harvard.atlassian.net/browse/ETD-161#icft=ETD-161


# What does this Pull Request do?
This sends marcxml records to the alma dropbox

# How should this be tested?

1) get alma_dropbox_id_rsa from vault and put in ./ssh/alma_dropbox_id_rsa
then get known_hosts from vault and put into .ssh/known_hosts 
2) copy .env.example to .env and set to dev settings
3) start up container
4) enter container and run pytest. all tests should pass.  
5) to test sftp, run  `python3 scripts/invoke-task.py` to upload
6) jaeger will confirm that those methods ran.

this has been deployed to dev.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? y
- integration tests? y

# Interested parties
Tag (@ mention) interested parties: @awoods
